src/Symfony/Component/Finder/Tests/FinderTest.php

```diff
@@ -267,13 +289,21 @@ public function testSize()
         $finder = $this->buildFinder();
         $this->assertSame($finder, $finder->files()->size('< 1K')->size('> 500'));
         $this->assertIterator($this->toAbsolute(array('test.php')), $finder->in(self::$tmpDir)->getIterator());
+
+        $finder = $this->buildFinder();
+        $this->assertSame($finder, $finder->files()->size(array('< 1K', '> 500')));
+        $this->assertIterator($this->toAbsolute(array('test.php')), $finder->in(self::$tmpDir)->getIterator());
     }
 
     public function testDate()
     {
         $finder = $this->buildFinder();
         $this->assertSame($finder, $finder->files()->date('until last month'));
         $this->assertIterator($this->toAbsolute(array('foo/bar.tmp', 'test.php')), $finder->in(self::$tmpDir)->getIterator());
+
+        $finder = $this->buildFinder();
```

Comment:
should be a separate test instead of making 2 independent tests in the same PHPUnit test. This makes things much better, as it means they will both run all the time (if the first logic fails, it does not prevent the second from running if they are in separate tests) and it also makes the test more readable (as each test stays self-contained)

Reply:
this is true for your other changes too.

And yes, I know that many Finder tests are still mixed in the same method. That's something we were doing a lot in the very early days of Symfony 2 development (i.e. in the early days of using PHPUnit in the Symfony community), and we haven't rewritten all existing tests to improve them.

Reply:
Thanks a lot for your explanations. 

I extracted my tests in their own methods but I might lack some inspiration to name them ...
What do you think about it ?

When I run php-cs-fix on tests, it throws an error 
`1) src/Symfony/Component/Finder/Tests/FinderTest.php (native_constant_invocation)`

I understand that this is a new way of writing native constant for more performance in Symfony. Is this something that I have to correct now?

I will be able to rewrite the old tests to update them by following the new practices in a dedicated PR.

Reply:
@jfredon currently, the automated CI checks are running v2.11 of php-cs-fixer. v2.12 has some new rules (see #27852 for the WIP about using them)